### PR TITLE
nested .forEach removed

### PR DESCRIPTION
**Pull request by grzim**
Filtering array each time an image is clicked can be replaced by a toggle function.

### CODE REVIEW

File: src/js/about-gallery.js [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+  let activeImage = galleryImages.find(image => image.classList.contains('gallery__image--active'));
 
   galleryImages.forEach(image => {
     image.addEventListener('mouseover', e => {
-      activateGalleryImage(e.target);
-
-      galleryImages
-        .filter(image => image !== e.target)
-        .forEach(image => deactivateGalleryImage(image));
+      toggleGalleryImage(activeImage, e.target)
```
Comment (typeofweb) [inline src/js/about-gallery.js]: Could you add `const nextImage = e.target;` here and not use `e.target` below?
Thanks!

Comment (typeofweb) [reply src/js/about-gallery.js]: Or maybe think of better name than nextImage, up to you really

**Review by typeofweb — CHANGES_REQUESTED** [review]

File: src/js/about-gallery.js (hunk 1 above)
Comment (lukaszklis) [reply src/js/about-gallery.js]: :+1: for `nextImage`

Comment (grzim) [reply src/js/about-gallery.js]: how about aliasing e.target in destructuring instead of introducing a variable?

**Pull request merged**